src/screens/CreatePost/index.js

```diff
@@ -60,7 +62,6 @@ import {
 } from '../../utils/constants';
 import {PROFILE_CACHE} from '../../utils/cache/constant';
 import {ShowingAudience, createPost} from '../../service/post';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `composeImageMeta` from '../../utils/string/file' has been removed. If this function is no longer needed, this change is fine. Otherwise, it might cause errors where the function is used.
```diff
- import {composeImageMeta} from '../../utils/string/file';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->